design-document.md

```diff
@@ -89,12 +94,11 @@ Simple Plan Budgeting App will provide a main web page providing a variety of op
 _Define the data models your service will expose in its responses via your *`-Model`* package. These will be equivalent to the *`PlaylistModel`* and *`SongModel`* from the Unit 3 project._
 
 ```
-
-
+```
```

Comment:
Take a look at how this file renders (not in the "code" view). I think you want to put your models inside of the triple backtick - as-is it's just rendering them all on one line and is confusing to read.